source/extensions/pageantjacker/pageantjacker.h

```diff
@@ -0,0 +1,55 @@
+/*!
+ * @file pageantjacker.h
+ * @brief Entry point and intialisation declrations for the pageantjacker extention.
+ */
+#ifndef _METERPRETER_SOURCE_EXTENSION_PAGEANTJACKER_PAGEANTJACKER_H
+#define _METERPRETER_SOURCE_EXTENSION_PAGEANTJACKER_PAGEANTJACKER_H
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+// TLVs
+// Output
+
+
+#define TLV_TYPE_EXTENSION_PAGEANTJACKER 0
+
+#define TLV_TYPE_EXTENSION_PAGEANTJACKER_STATUS		  MAKE_CUSTOM_TLV(TLV_META_TYPE_BOOL, TLV_TYPE_EXTENSION_PAGEANTJACKER, TLV_EXTENSIONS + 1)
+#define TLV_TYPE_EXTENSION_PAGEANTJACKER_ERRORMESSAGE MAKE_CUSTOM_TLV(TLV_META_TYPE_STRING, TLV_TYPE_EXTENSION_PAGEANTJACKER, TLV_EXTENSIONS + 2)
+#define TLV_TYPE_EXTENSION_PAGEANTJACKER_RETURNEDBLOB MAKE_CUSTOM_TLV(TLV_META_TYPE_RAW, TLV_TYPE_EXTENSION_PAGEANTJACKER, TLV_EXTENSIONS + 3)

```

Comment:
can probably shorten `PAGEANTJACKER` to `PJACK` to make things a bit smaller. Also need to sort out alignment a bit here (use spaces not tabs).